src/main/java/oscar/login/jaas/OscarGroup.java

```diff
@@ -56,25 +55,44 @@
         this.name = name;
     }
 
-    @Override
+    /**
+     * Adds a member to the group.
+     *
+     * @param user the Principal to be added
+     * @return true if the member was added, false if already exists
+     */
     public boolean addMember(Principal user) {
         if (!principals.contains(user))
             principals.add(user);
         return true;
     }
 
-    @Override
+    /**
+     * Removes a member from the group.
+     *
+     * @param user the Principal to be removed
+     * @return true if the member was removed, false otherwise
```

Comment:
again -

Reply:
Fixed the documentation.

Reply:
sourcery don't change your behaviour.  This was important.  Please be sure to comment on these issues.

Reply:
at best, make an issue ticket for this for further investigation and share with magenta.  Good to update the docs but if it said it should work a particular way that is likely important.

keep the docs as they were but add an addendum if you agree with sourcery on the actual behaviour

Reply:
I understand Sourcery's suggestions, but I'm just wondering why the previous development team has set up the logic that way. It might be good to return false when a member doesn't exist (same as the last issue, return false when a member already existed), but shall we discuss with Magenta before modifying the code?